godal.go

```diff
@@ -2627,13 +2674,117 @@ func (f *Feature) SetGeometry(geom *Geometry, opts ...SetGeometryOption) error {
 	return cgc.close()
 }
 
+//SetGeometryColumnName set the name of feature first geometry field.
+func (f *Feature) SetGeometryColumnName(name string) {
+	cname := C.CString(name)
+	defer C.free(unsafe.Pointer(cname))
+	gfdef := C.OGR_F_GetGeomFieldDefnRef(f.handle, C.int(0))
+	if gfdef != nil {
+		C.OGR_GFld_SetName(gfdef, (*C.char)(unsafe.Pointer(cname)))
+	}
+}
+
+//SetFID set feature identifier
+func (f *Feature) SetFID(fid int) {
+	// OGR error returned is always none, so we don't handle it
+	C.OGR_F_SetFID(f.handle, C.GIntBig(fid))
+}
+
+//SetFieldValue set feature's field value
+func (f *Feature) SetFieldValue(field Field, value interface{}) error {
```

Comment:
This function needs to be guarded by the standard godal error handler in order to catch warnings/errors raised by incompatible values

Reply:
handler added